lib/manager/poetry/artifacts.ts

```diff
@@ -76,8 +109,11 @@ export async function updateArtifacts({
     const tagConstraint = getPythonConstraint(existingLockFileContent, config);
     const poetryRequirement = config.compatibility?.poetry || 'poetry';
     const poetryInstall = 'pip install ' + quote(poetryRequirement);
+    const sourceCredentialVars = await getSourceCredentialVars(packageFileName);
```

Comment:
Pass `newPackageFileContent` instead, it is the contents of the `pyproject.toml`

Reply:
:facepalm: Thanks -- didn't realize that!